app/javascript/src/components/Notes/Pane/Form.jsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+
+import { Form, Formik } from "formik";
+import { Check } from "neetoicons";
+import { Pane, Toastr } from "neetoui";
+import { Button, Input, Select } from "neetoui/formik";
+
+import {
+  NOTES_FORM_CONTACT_DATA,
+  NOTES_FORM_INITIAL_VALUES,
+  NOTES_FORM_TAG_DATA,
+  NOTES_FORM_VALIDATION_SCHEMA,
+} from "components/Notes/constants";
+
+const CreateForm = ({ setShowPane }) => (
+  <Formik
+    initialValues={NOTES_FORM_INITIAL_VALUES}
+    validationSchema={NOTES_FORM_VALIDATION_SCHEMA}
+  >
+    <Form>
+      <Pane.Body>
+        <Input
+          required
+          className="mt-5 w-full flex-grow-0"
+          label="Title"
+          name="title"
+        />
+        <Input
+          required
+          className="mt-5 w-full flex-grow-0"
+          label="Description"
+          name="description"
+        />
+        <Select
+          isClearable
+          isSearchable
+          required
+          className="mt-5 w-full flex-grow-0"
+          label="Assigned Contact"
+          name="assignedContact"
+          options={NOTES_FORM_CONTACT_DATA}
+          placeholder="Select Role"
```

Comment:
Label says `contact` but placeholder says `role`. How does that make sense?

Reply:
Was using the text provided in the figma design , will change it.

Reply:
It's possible that when the client hands you off with the design, there are some disparities here and there. Thus rather than blindly developing what's shown in the design, you should be verifying what you are creating and whether it makes sense before even developing it. In case of any confusions, you should clarify it first. If you feel that it was a mistake in the design, then you should build it right during development and then later point out the change in PR.